app/controllers/register/trustees/AddTrusteeController.scala

```diff
@@ -53,9 +54,15 @@ class AddTrusteeController @Inject()(
   def onPageLoad(mode: Mode): Action[AnyContent] = (authenticate andThen getData andThen requireData).async {
     implicit request =>
       SchemeDetailsId.retrieve.right.map { schemeDetails =>
-        val trustees = request.userAnswers.allTrusteesAfterDelete
 
-        Future.successful(Ok(addTrustee(appConfig, form, mode, trustees)))
+        val listOfSchemeTypeTrusts: Seq[SchemeType] = Seq(SchemeType.SingleTrust, SchemeType.MasterTrust)
```

Comment:
Make a def or a lazy val so won't be instantiated if hub off

Reply:
fixed